scm/driver/stash/repo.go

```diff
@@ -192,6 +192,18 @@ func (s *repositoryService) ListV2(ctx context.Context, opts scm.RepoListOptions
 	return convertRepositoryList(out), res, err
 }
 
+// ListNamespace returns the user repository list based on searchterm and namespace.
+func (s *repositoryService) ListNamespace(ctx context.Context, namespace string, opts scm.ListOptions) ([]*scm.Repository, *scm.Response, error) {
+	path := fmt.Sprintf("rest/api/1.0/projects/%s/repos?%s", namespace, encodeListRoleOptions(opts))
+	out := new(repositories)
+	res, err := s.client.do(ctx, "GET", path, nil, &out)
+	if res != nil && !out.pagination.LastPage.Bool {
+		res.Page.First = 1
+		res.Page.Next = opts.Page + 1
```

Comment:
this differes from the above, should be `opts.ListOptions.Page + 1`?

Reply:
@johannesHarness this is copied from List call. ListV2 has addition of search functionality which doesn't make sense here.

Reply:
ohh didn't expand that much. makes sense, thanks